src/pages/documentation/profile/index.md

```diff
@@ -1,80 +0,0 @@
-import Tabs from './tabs/index.md'
-
-# Profile
```

Comment:
The Profile extension should be kept also since it has an ACP version. Make sure it is displayed in summary (gatsby-config) and it contains all the required files as the other extensions